src/Common/src/TypeSystem/Common/DefType.FieldLayout.cs

```diff
@@ -230,6 +230,14 @@ public int ThreadStaticFieldAlignment
             }
         }
 
+        /// <summary>
+        /// Do the fields of the type satisfy the Homogeneous Float Aggregate classification on ARM architecture?
+        /// </summary>
+        public virtual bool IsHFA()
```

Comment:
Since this is not overriden anywhere, it's hard to comment on whether this should or shouldn't be on the core type system. Cc @davidwrighton